Library/Formula/gdk-pixbuf.rb

```diff
@@ -21,6 +21,9 @@ class GdkPixbuf < Formula
   depends_on "libpng"
   depends_on "gobject-introspection"
 
+  # disable the default relocatable library setting
+  patch :DATA
```

Comment:
Alright. I don't mind carrying a short-term patch, but anything reasonably simple like this that can be `inreplace`'d should be if it's going to stick around for a while. The less in-formula patches the better; they've a habit of breaking :cry: